### ISSUE TITLE
Поле ввода e-mail для подписки в футере нефункционально

### ISSUE THREAD

**Issue by Yamortis** [Yamortis/Test_health.mail.ru#3]
**Предусловие:**
1. Залогиниться на mail.ru под своим аккаунтом.
2. Открыть в браузере страницу сервиса https://health.mail.ru/

**Шаги:**
1. Скроллом или бегунком пролистать страницу вниз до формы подписки https://yadi.sk/i/EwIPfvLyk5f5uA
2. Поместить курсор в область поля для текста и попытаться ввести другой e-mail адрес вместо уже подставленного вашего.

**Фактический результат:**
поле неактивно (курсор мыши по клику на него не превращается в текстовый курсор), ввести другой адрес нельзя.

**Ожидаемый результат:**
курсор мыши по клику на текстовом поле превращается в текстовый курсор, можно ввести другой адрес.

3. Разлогиниться.
4. Повторить шаги 1 и 2.

**Фактический результат:**
поле неактивно (курсор мыши по клику на него не превращается в текстовый курсор), в поле находится неизменяемая надпись "Авторизуйтесь для подписки", ввести адрес нельзя.

**Ожидаемый результат:**
курсор мыши по клику на текстовом поле превращается в текстовый курсор, можно ввести e-mail адрес.

**Comment by Svetlana678** [comment]
Да, ошибка, поле действительно неактивно. По оформлению, мне кажется, было бы достаточно оставить только первый фактический результат и ожидаемый. Два ожидаемых и фактических результата к одному багу создают небольшую путаницу, к одной ошибке должен быть один фактический, один ожидаемый результат. Поэтому лучше обрезать описание после первого ожидаемого результата и создать еще один баг уже с вариантом 2 (разлогиниться, авторизоваться для подписки).

**Comment by OlgaKondratieva** [comment]
Поле подписки настроено так, что когда Вы авторизуетесь там автоматически появляется адрес, чтобы осталось только нажать кнопку "подписаться" или "отписаться". Оно не предусмотрено для того чтобы ввести другой адрес.

**Comment by Svetlana678** [comment]
Возможно, но по условиям задания, нам не представлены спецификации или технические условия, где это было бы описано, поэтому как мы можем знать наверняка,что предусмотрено?
